Adding SkyPilot example for FlexGen (#1)
Michaelvll/FlexGen: README.md

```
Note that only a subset of HELM scenarios is tested. See more tested scenarios [here](flexgen/apps/helm_passed_30b.sh).

### Run FlexGen on Any Cloud with SkyPilot
FlexGen benchmark can be launched with [SkyPilot](https://github.com/skypilot-org/skypilot), a tool for launching ML jobs on any cloud.
First, install SkyPilot and check you have some cloud credentials ([docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)):
```bash
pip install "skypilot[aws,gcp,azure,lambda]"  # pick your clouds
sky check
```

> You can now use a single command to automatically launch the benchmark on any cloud:

Reviewer: ```suggestion
You can now use a single command to launch the benchmark on any cloud, which automatically finds a region (in the cheapest-price order) with availability for the requested GPUs:
```

```
Note that only a subset of HELM scenarios is tested. See more tested scenarios [here](flexgen/apps/helm_passed_30b.sh).

> ### Run FlexGen on any cloud with SkyPilot

Reviewer: nit: Capital Case to be consistent with this doc?

Author: Good catch! Done. Thanks!

> FlexGen benchmark can be launched with [SkyPilot](http://skypilot.co), a tool for launching ML jobs on any cloud.

Reviewer: ```suggestion
FlexGen benchmark can be launched with [SkyPilot](https://github.com/skypilot-org/skypilot), a tool for launching ML jobs on any cloud.
```

> You can use a single command below to automatically launch the benchmark on any cloud with SkyPilot, after you setup your cloud account locally (check how to setup SkyPilot [here](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)).

Reviewer: How about:

First, install SkyPilot and check you have some cloud credentials ([docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html#cloud-account-setup)):
```bash
pip install "skypilot[aws,gcp,azure,lambda]"  # pick your clouds
sky check
```
You can now use a single command to automatically launch the benchmark on any cloud:

Author: Good point! Replaced. Thanks!

> ```

Reviewer: ```suggestion
```bash
```

Author: Done. Thanks!

> sky launch -c flexgen --detach-setup flexgen/apps/task.yaml

Reviewer: ```suggestion
sky launch -c flexgen --detach-setup --down flexgen/apps/task.yaml
```

```

> Note that you can replace the run section with any FlexGen command. You can log into the cluster running the job with `ssh flexgen` and terminate the cluster with `sky down flexgen`.

Reviewer: ```suggestion
You can then log into the cluster running the job with `ssh flexgen` for monitoring. Once the job has finished, the cluster will be automatically terminated due to the `--down` flag.

To run any other FlexGen command, you can edit [`flexgen/apps/task.yaml`](./flexgen/apps/task.yaml) and replace the `run` section.
```

Reviewer: I added `--down` and a sentence to explain it. Wdyt? We can also keep the original version of manually running `sky down`. Pros are that the job seems to run pretty long, so people may want to ctrl-c in the middle and manually terminate it. Using autodown showcases a good feature, however.

Author: How about we keep `--down` in the text instead of in the command as following:
```
You can then log into the cluster running the job with `ssh flexgen` for monitoring. Once the job has finished, you can terminate the cluster with `sky down flexgen` or pass in `--down` flag to the command above to have the cluster terminate itself automatically.
```

Reason:
1. With `--down` and if the user detaches from the log, they will never be able to find the log after the cluster is automatically terminated.
2. Adding `--down` makes the launching command longer, which may not look good.

Open to discussions : )

Reviewer: Makes sense. I think either the current version or the original `sky down` version is fine, up to you.
